packages/office-ui-fabric-react/src/components/Popup/Popup.tsx

```diff
@@ -115,8 +115,20 @@ export class Popup extends BaseComponent<IPopupProps, IPopupState> {
   private _getScrollBar() {
     let needsVerticalScrollBar = false;
     if (this._root && this._root.value && this._root.value.firstElementChild) {
-      needsVerticalScrollBar = this._root.value.clientHeight > 0
-          && this._root.value.firstElementChild.clientHeight > this._root.value.clientHeight;
+      // ClientHeight returns the client height of an element rounded to an
+      // integer. On some browsers at different zoom levels this rounding
+      // can generate different results for the root container and child even
+      // though they are the same height. This causes us to show a scroll bar
+      // when not needed. Ideally we would use BoundingClientRect().height
+      // instead however seems that the API is 90% slower than using ClientHeight.
+      // Therefore instead we will calculate the difference between heights and
+      // allow for a 1px difference to still be considered ok and not show the
+      // scroll bar.
+      const rootHeight = this._root.value.clientHeight;
+      const firstChildHeight = this._root.value.firstElementChild.clientHeight;
+      if (rootHeight > 0 && firstChildHeight > rootHeight) {
+        needsVerticalScrollBar = (firstChildHeight - rootHeight) > 1;
```

Comment:
what if rootHeight is < 0?

Reply:
It looks like you could get into a situation where the logic is different now than it used to be

Reply:
so I am leaving that as it was, which it just does not show the scrollbar, not sure when that happens

---
In reply to: [181263841](https://github.com/OfficeDev/office-ui-fabric-react/pull/4528#discussion_r181263841) [](ancestors = 181263841)